patcher.sh

```diff
@@ -19,8 +19,20 @@ VERSION="1.5.5"
 GAMEDIR="${SCRIPT_DIR}/am2r_${VERSION}"
 RESOURCES="${GAMEDIR}/assets"
 
+# Check wether a command is installed
+checkInstalled() {
+        local command="$1"
```

Comment:
Formatting seems weird here.

Reply:
What would you prefer?

Reply:
![image](https://user-images.githubusercontent.com/38186597/184553553-5e17ca2a-3d01-4538-abb7-c8348bf268a5.png)
the variable assignment and comment should be aligned with the if condition.

Reply:
Should be good now :+1: